pkg/manager/member/template.go

```diff
@@ -49,11 +49,24 @@ then
 fi
 
 # Use HOSTNAME if POD_NAME is unset for backward compatibility.
-POD_NAME=${POD_NAME:-$HOSTNAME}
+POD_NAME=${POD_NAME:-$HOSTNAME}{{ if .FormatClusterDomain }}
+pd_url="{{ .Path }}"
+encoded_domain_url=` + "`" + `echo $pd_url | base64 | tr "\n" " " | sed "s/ //g"` + "`" + `
```

Comment:
can we use `$(echo $pd_url | base64 | tr "\n" " " | sed "s/ //g")` here to avoid the **`**?

Reply:
Good point!